src/main/java/by/exadel/internship/service/impl/UserServiceImpl.java

```diff
@@ -149,4 +153,14 @@ public void updateTimeSlot(UUID userId, List<UserTimeSlotDTO> userTimeSlotDTOLis
         return mapper.map(users);
     }
 
+    @Override
+    public Set<UserDTO> getUsersBySkills(List<Skill> skills) {
+        List<String> skillsToString = skills.stream().map(Enum::toString).collect(Collectors.toList());
```

Comment:
Minor: each method call from new line, please

Reply:
This line I dropped. But I got you, thanks

Reply:
I got it back and redone.